packages/core/src/shipping/strategies/paypal-commerce/paypal-commerce-fastlane-shipping-strategy.ts

```diff
@@ -75,7 +71,7 @@ export default class PayPalCommerceFastlaneShippingStrategy implements ShippingS
 
                 this._isFastlaneEnabled = !!paymentMethod.initializationData.isFastlaneEnabled;
 
-                await this._initializePayPalSdk(methodId, styles);
+                await this._initializePayPalSdk(methodId, paypalcommercefastlane?.styles);
                 await this._runAuthenticationFlowOrThrow(methodId);
```

Comment:
Can we to `Promise.all` so we are not calling these in sequence but in parallel.

Reply:
We should not call this in parallel, because _runAuthenticationFlowOrThrow should be called only after PayPal Sdk initialization